### PR TITLE
Fix/taskboard fix auth

### PR DESCRIPTION
**Pull request by EcCo18**

### CODE REVIEW

File: backend/burning-okr/burning-okr-logic/src/main/java/org/burningokr/service/activity/ActivityService.java [hunk 1]
```diff
@@ -31,9 +27,9 @@ public ActivityService(AuthorizationUserContextService authorizationUserContextS
    * @param <T>    generic Type extends Trackable Long
    */
 
-  //FIXME: activity is not created, to update the task in the taskboard
+  //FIXME: (C.C.) activity is not created, to update the task in the taskboard
```
Comment (MeepMr) [inline backend/burning-okr/burning-okr-logic/src/main/java/org/burningokr/service/activity/ActivityService.java:30]: TODO

File: backend/burning-okr/burning-okr-app/src/main/java/org/burningokr/config/WebsocketAuthenticationChannelInterceptor.java [hunk 2]
```diff
@@ -29,47 +32,58 @@ public Message<?> preSend(Message<?> message, MessageChannel channel) {
...
 
+  /**
+   * checks if message is initial connection attempt
+   * @return true if connection attempt
+   */
   private boolean isConnectionAttempt(StompHeaderAccessor accessor) {
     return accessor != null && StompCommand.CONNECT.equals(accessor.getCommand());
   }
 
+  // TODO fix auth
```
Comment (lionelpa) [inline backend/burning-okr/burning-okr-app/src/main/java/org/burningokr/config/WebsocketAuthenticationChannelInterceptor.java]: Remove

File: backend/burning-okr/burning-okr-logic/src/main/java/org/burningokr/service/monitoring/MonitorService.java [hunk 3]
```diff
@@ -9,48 +8,35 @@
...
     if (!monitoringUsers.containsKey(monitoredObject)) {
-      monitoringUsers.put(monitoredObject, Collections.synchronizedList(new LinkedList<>()));
+      monitoringUsers.put(monitoredObject, Collections.synchronizedList(new LinkedList<UUID>()));
     }
-    List<UserId> currentMonitors = monitoringUsers.get(monitoredObject);
-    boolean newUser = !monitoringUsers.get(monitoredObject).contains(userId);
-    currentMonitors.add(userId);
-    return true; // TODO: Rückgabe von newUser, aber muss aktuell immer raus, sonst erhält unter gewissen Umständen der Benutzer in einem weiteren Tab die Liste nicht. (MV)
+    List<UUID> currentMonitoredUsers = monitoringUsers.get(monitoredObject);
+    boolean newUser = !monitoringUsers.get(monitoredObject).contains(user.getId()); // TODO (C.K.): check why
```
Comment (lionelpa) [inline backend/burning-okr/burning-okr-logic/src/main/java/org/burningokr/service/monitoring/MonitorService.java]: check if todo is resolved

File: backend/burning-okr/burning-okr-logic/src/main/java/org/burningokr/service/monitoring/MonitorService.java [hunk 4]
```diff
@@ -9,48 +8,35 @@
...
+    currentMonitoredUsers.add(user.getId());
+
+    // TODO: Rückgabe von newUser, aber muss aktuell immer raus, sonst erhält unter gewissen Umständen
```
Comment (lionelpa) [inline backend/burning-okr/burning-okr-logic/src/main/java/org/burningokr/service/monitoring/MonitorService.java]: check if todo is resolved

**Review by MeepMr — APPROVED** [review]

**Pull request merged**